daemons/gptp/common/ieee1588port.cpp

```diff
@@ -1274,17 +1276,20 @@ void IEEE1588Port::becomeSlave( bool restart_syntonization ) {
 
 	port_state = PTP_SLAVE;
 
-	/* TODO : Seems this be active here
+	/* TODO : Should this be active for non-automotive profile uses
```

Comment:
These commented out things relating to starting the sync timeout are apparently from [this commit](https://github.com/AVnu/Open-AVB/commit/32d59d82495d2aabda9c952016a6a2dce372be90l), which is supposed to wait for the first sync receive to start.  There is indeed code to start the SYNC_RECEIPT timer when a Sync Follow-Up is received by a clock slave.

I'm not sure this is technically correct, but it would take a strangely malfunctioning master for it to cause a problem and it's probably convenient in some cases where you want to force a device to wait for the master to arrive rather than take over as temporary master.

We should probably delete the commented out code at some point, unless someone objects to this behavior.